nucypher/cli/deploy.py

```diff
@@ -209,16 +210,16 @@ def deploy(click_config,
         click.clear()
         click.secho(NU_BANNER)
 
-        w3 = deployer.blockchain.interface.w3
+        w3 = deployer.blockchain.w3
         click.secho(f"Current Time ........ {maya.now().iso8601()}")
-        click.secho(f"Web3 Provider ....... {deployer.blockchain.interface.provider_uri}")
+        click.secho(f"Web3 Provider ....... {deployer.blockchain.provider_uri}")
         click.secho(f"Block ............... {w3.eth.blockNumber}")
```

Comment:
Another client function perhaps?

Reply:
Actually, that one is not an RPC endpoint.